src/client/js/WebGME.js

```diff
@@ -89,8 +89,10 @@ define([
             // URL query has higher priority than the config.
             if ((initialThingsToDo.projectToLoad || initialThingsToDo.createNewProject) === false) {
                 initialThingsToDo.projectToLoad = gmeConfig.client.defaultProject.name;
-                initialThingsToDo.branchToLoad = initialThingsToDo.branchToLoad || gmeConfig.client.defaultProject.branch;
-                initialThingsToDo.objectToLoad = initialThingsToDo.objectToLoad || gmeConfig.client.defaultProject.node;
+                initialThingsToDo.branchToLoad = initialThingsToDo.branchToLoad ||
+                    gmeConfig.client.defaultProject.branch;
+                initialThingsToDo.objectToLoad = initialThingsToDo.objectToLoad ||
+                    gmeConfig.client.defaultProject.node || initialThingsToDo.objectToLoad; // i.e. the root-node.
```

Comment:
I think it is still not right.
First of all, if you have anything in the config, then it will override the ROOT path.
Secondly, the intialThingsToDo.objectToLoad already 'defaulted' the node path to ROOT path, which makes the whole config option meaningless in this form.
-> we either loose this config option, or we not default the param from the URL, and then the expression could be URL || config || ROOT